app/connector/email/src/main/java/io/syndesis/connector/email/meta/EMailMetadata.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.connector.email.meta;
+
+import io.syndesis.connector.email.EMailConstants;
+
+public class EMailMetadata implements EMailConstants {
```

Comment:
```suggestion
public class EMailMetadata {
```

nit: doesn't seem to be used

Reply:
Protocol comes from EMailConstants